--- cmt_1.17/src/Makefile.orig	2019-04-09 19:07:19.010095393 +0200
+++ cmt_1.17/src/Makefile	2019-04-09 19:08:13.183135246 +0200
@@ -11,7 +11,7 @@
 # GENERAL
 #
 
-CFLAGS		=	$(INCLUDES) -Wall -Werror -O2 -fPIC
+CFLAGS		=	$(INCLUDES) -Wall -Werror -fPIC $(OPTFLAGS)
 CXXFLAGS	=	$(CFLAGS)
 PLUGIN_LIB	=	../plugins/cmt.so
 
@@ -76,7 +76,7 @@
 		$(PLUGIN_OBJECTS)					
 
 install:	$(PLUGIN_LIB)
-	cp $(PLUGIN_LIB) $(INSTALL_PLUGINS_DIR)
+	cp $(PLUGIN_LIB) $(DESTDIR)$(INSTALL_PLUGINS_DIR)
 
 test:	/tmp/test.wav ../../ladspa_sdk/snd/noise.wav always
 	@echo ---------------------------------------------
